apps/web/pages/organizations/[id]/invoiceTable.tsx

```diff
@@ -0,0 +1,53 @@
+import { FragmentType, graphql, useFragment } from '../../../frontend/generated/gql'
+
+export const InvoiceTableItemFragment = graphql(`
+  fragment InvoiceTableItem on Invoice {
+    id
+    invoiceDate
+    customerName
+    items {
+      id
+      duration
+      hourlyRate
+    }
+  }
+`)
+
+interface InvoiceTableProps {
+  invoices: FragmentType<typeof InvoiceTableItemFragment>[]
+}
+
+export const InvoiceTable = (props: InvoiceTableProps): JSX.Element => {
+  const invoices = useFragment(InvoiceTableItemFragment, props.invoices)
+
+  return (
+    <div>
+      <table className="table">
+        <thead className="text-lg text-base-content">
+          <tr>
+            <th>Date</th>
+            <th>Customer</th>
+            <th>Hours</th>
+            <th>Total amount</th>
+          </tr>
+        </thead>
+        <tbody className="text-base">
+          {invoices.map((invoice) => (
+            <tr key={invoice.id}>
+              <td>{invoice.invoiceDate}</td>
+              <td>{invoice.customerName}</td>
+              <td>{invoice.items.reduce((sum, item) => sum + item.duration, 0)}</td>
+              <td>
+                {invoice.items.some(
+                  (item) => item.hourlyRate === null || item.hourlyRate === undefined || item.hourlyRate === 0,
+                )
+                  ? '-'
+                  : invoice.items.reduce((sum, item) => sum + item.duration * item.hourlyRate!, 0)}
```

Comment:
```suggestion
            <td>{invoice.items.reduce((sum, item) => sum + item.duration * item.hourlyRate, 0)}</td>
```
I thought this because `hourlyRate` is not gonna be nullable.

Reply:
Not yet implemented, which causes following error.. Should we still go through with it?
![image](https://github.com/user-attachments/assets/44be9447-6000-4b8b-a41e-354564777e50)

Reply:
I tried it now and it works.